cypress/e2e/notifications.spec.ts

```diff
@@ -82,39 +100,66 @@ describe("notifications", () => {
   })
   it("should send out a notification to reviewers when someone edits the site while a PR is open", () => {
     // Arrange
+    cy.setEmailSessionDefaults("Email collaborator")
+    visitE2eEmailTestRepo()
+    api.editUnlinkedPage(
+      "privacy.md",
+      genRandomString(),
+      E2E_EMAIL_TEST_SITE.repo
+    )
     // NOTE: Editor should not see a notification
+    getNotificationsButton().click()
+    getNotificationList().should(
+      "have.css",
+      "background-color",
+      READ_COMMENTS_BG_COLOR
+    )
     // NOTE: All other reviewers should see a notification
-    // Act
-    // Assert
-  })
-  it("should receive a notification on successful comment post", () => {
-    // Arrange
-    // Act
+    cy.setEmailSessionDefaults("Email admin")
+    visitE2eEmailTestRepo()
     // Assert
+    getNotificationsButton().click()
+    cy.contains(getChangesMadeNotifText(E2E_EMAIL_COLLAB.email))
+      .should("be.visible")
+      .should("have.css", "background-color", UNREAD_COMMENTS_BG_COLOR)
   })
   it("should receive a notification on approval of a review request", () => {
     // Arrange
+    cy.setEmailSessionDefaults("Email collaborator")
+    visitE2eEmailTestRepo()
+    api.approveReviewRequest(reviewId)
+
     // Act
+    cy.setEmailSessionDefaults("Email admin")
+    visitE2eEmailTestRepo()
+    getNotificationsButton().click()
+
     // Assert
-  })
-  it("should receive a notification on merge of review request", () => {
-    // Arrange
-    // Act
-    // Assert
-  })
```

Comment:
Just checking, was this test also deprioritised?

Reply:
![Screenshot 2023-07-28 at 3 01 19 PM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/238e2d0a-109e-4bd0-99a3-38ffa98afc9f)
there is no `merged` noti for now

Reply:
^ tis from our be uh